### ISSUE TITLE
Deveria ter feito como os colegas

### ISSUE THREAD

**Issue by rodrigonoll** [lab-ds/crm-lab-team#41]
https://github.com/lab-ds/crm-lab-team/blob/7369f29c148ee9402ef6877daa867da0f7db39a2/src/main/resources/templates/case/view.html#L97-L114

**Comment by FelipeEmerim** [comment]
Você diz de inserir as atividades? Pensei que isto ficaria para a próxima entrega.

**Comment by rodrigonoll** [comment]
Não, isso era dessa como tinha comentado na semana passada. Mas a data de entrega desses fixes é dia 20/5 e a nota das coisas que faltaram nessa entrega será para o T3

**Comment by FelipeEmerim** [comment]
Eu vi que ficou combinado que cada um deveria inserir sua parte na dos colegas é isso?

**Comment by rodrigonoll** [comment]
sim, cada um deve inserir o fragmento na funcionalidade dos colegas

--

Atenciosamente,

*Prof. Dr. Rodrigo Noll*
Professor de Engenharia de Software e Programação
Instituto Federal de Educação, Ciência e Tecnologia do Rio Grande do Sul -
IFRS - Campus Canoas
Rua Dra. Maria Zélia Carneiro de Figueiredo, 870-A
Bairro Igara III - CEP 92412-240 - Canoas/RS
Fone: (51) 3415.8200– Ramal: 220
E-mail: rodrigo.noll@canoas.ifrs.edu.br
Website: http://www.canoas.ifrs.edu.br



Em dom, 19 de mai de 2019 às 20:17, FelipeEmerim <notifications@github.com>
escreveu:

**Comment by FelipeEmerim** [comment]
Não acha que podemos fechar essa issue? Pois eles teriam que popular minha aba de atividades. Ou trocar o responsável
